test/room_test.rb

```diff
@@ -3,22 +3,26 @@
 require './lib/room'
 
 class RoomTest < Minitest::Test
+  def setup
+    @room_1 = Room.new(:bedroom, 10, '13')
+    @room_2 = Room.new(:living_room, 15, '12')
+  end
   def test_it_exists
-    room = Room.new(:bedroom, 10, '13')
-    assert_instance_of Room, room
+    assert_instance_of Room, @room_1
   end
 
   def test_it_has_a_category
-    skip
-    room = Room.new(:bedroom, 10, '13')
-    assert_equal :bedroom, room.category
+    assert_equal :bedroom, @room_1.category
   end
 
   def test_it_can_get_area
-    skip
-    room1 = Room.new(:bedroom, 10, '13')
-    room2 = Room.new(:living_room, 15, '12')
-    assert_equal 130, room1.area
-    assert_equal 180, room2.area
+    assert_equal 130, @room_1.area
+    assert_equal 180, @room_2.area
+  end
+
+  def test_is_painted_and_paint
+    refute @room_1.is_painted?
+    @room_1.paint
+    assert @room_1.is_painted?
```

Comment:
Setup methods can save time, but in this case, rewriting what was already there to use a setup probably cost you more time than it saved, since you only needed to write one test of your own.  Only do setup methods when they make sense.